tests/unit/maxmemory.tcl

```diff
@@ -74,7 +74,7 @@ start_server {tags {"maxmemory" "external:skip"}} {
             }
 
             verify_test $client_eviction
-        }
+        } {} {tls:skip}
```

Comment:
Maybe we better add a comment as to why it can't be used with TLS.

Reply:
:+1: